### ISSUE TITLE
Ghost measure number labels

### ISSUE THREAD

**Issue by nikobeer** [Edirom/Edirom-Online#72]
After "some" time of navigating through sources, measure number labels from previously viewed measures won't disappear.

![ghost_measures](https://cloud.githubusercontent.com/assets/2676362/10308963/e2fe52a8-6c39-11e5-9065-0e1da00ffbfd.png)

**Comment by bwbohl** [comment]
same problem here

**Comment by bwbohl** [comment]
seems to be connected to digilib suspending service
<html>
<head>
<meta http-equiv="Content-Type" content="text/html;charset=utf-8"/>
<title>Error 503 Service Unavailable</title>
</head>
<body><h2>HTTP ERROR 503</h2>
<p>Problem accessing /digilib/Scaler/OMITTED/01_062_0064_005_01.jpg. Reason:
<pre>    Service Unavailable</pre></p><hr><a href="http://eclipse.org/jetty">Powered by Jetty:// 9.4.0.v20161208</a><hr/>

</body>
</html>

**Comment by bwbohl** [comment]
actually not really sure of my previous comment but, at least it's more likely to happen when using concordance navigator and switching measures across pages really fast…

**Comment by bwbohl** [comment]
could be connected to task management

**Comment by krHERO** [comment]
@bwbohl @nikobeer is this still an issue? thanks for feedback

**Comment by nikobeer** [comment]
Yes and no. I think we have solved it in RWA. @lexngu could we have a look into this? I think we dealt with this problem – again – while working on https://github.com/Max-Reger-Institut/Edirom-Online/issues/27.

**Comment by lexngu** [comment]
This could be related to https://github.com/Max-Reger-Institut/Edirom-Online/issues/22 : Elements were created which already existed; I think the solution is implemented by these lines: https://github.com/Max-Reger-Institut/Edirom-Online/blob/6e9e9d768bf9e2a4758d22915e315e83fe6ac685/app/view/window/image/ImageViewer.js#L208-L209

**Comment by bwbohl** [comment]
Thanks for reporting back. We should compare your referenced code to the one in this repo to decide.

**Comment by bwbohl** [comment]
@lexngu & @nikobeer could you please run the comparison and some manual test on this?
